documentation/api/query/v4/operators.markdown

```diff
@@ -176,6 +178,30 @@ To get the average uptime for your nodes,
 
     ["extract", [["function", "avg", "value"]], ["=", "name", "uptime_seconds"]]
 
+## Context Operators
+
+*Note:* Context setting support is new and experimental. Setting the context at
+the top of the query is only supported on the [root] endpoint.
+
+Setting context in a query allows you to choose the entity you are querying
+on. This augments the endpoint support we have today, whereby the endpoint
+decides the context by allowing a single endpoint to contain all the information
```

Comment:
I think you can cut off this sentence and just say "whereby the endpoint decides the context. For example `/pdb/query/v4/nodes` sets the context of the query to `nodes`.